parl/remote/tests/send_job_test.py

```diff
@@ -51,13 +47,12 @@ def test_send_file(self):
         parl.connect(
             'localhost:{}'.format(port),
             distributed_files=['./rom_files/pong.bin'])
-        time.sleep(3)
+        time.sleep(5)
```

Comment:
If you find it hard to determine the sleep time, please use multiple try here, just like what you have done before.